src/scripts/main.js

```diff
@@ -1,3 +1,29 @@
 'use strict';
 
-// write code here
+const employees = document.querySelectorAll('li');
+
+const sortList = function([...list]) {
+  list.sort(
+    (worker1, worker2) => salaryToNumber(worker2.dataset.salary)
+    - salaryToNumber(worker1.dataset.salary));
+
+  list.forEach(worker => document.querySelector('ul').append(worker));
+};
+
+function salaryToNumber(salary) {
+  return salary.slice(1).split(',').join('');
+}
```

Comment:
It is a good tone to declare a function before use.
Btw, method join() returns a string, so the name of the function is a bit confusing